js/foundation.util.touch.js

```diff
@@ -140,15 +140,10 @@ class Touch {}
   };
 
   Touch.init = function($) {
```

Comment:
I think @gakimball is right, this is really an object literal...

These touch handlers are actually one of the more awkward feeling areas in the codebase to me. I'm leaning towards killing this and adding a hammer.js dependency (as a separate effort from this PR)